pdns/dnsname.hh

```diff
@@ -299,6 +299,11 @@ struct SuffixMatchTree
    */
   void remove(std::vector<std::string> labels) const
   {
```

Comment:
Not related to this dif, but would it be better to make the argument type `const std::vector<std::string> &`?